### PR TITLE
SOPs for storageclass, ingresscontroller and infra node rebalance.

### PR DESCRIPTION
**Pull request by k-wall**
## What

Adds SOPs for:

* creation of storageclass for managed kafka
* creation of ingresscontroller for managed kafka
* resolving if the infra node rebalance.

## Why

Supports new service for 90-day goal.

## How

SOPs written.

### CODE REVIEW

**Comment by k-wall** [comment]
@david-martin as requested, hopefully.

File: sops/sharded_ingresscontroller.asciidoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+domain=$(oc get ingresscontroller -n openshift-ingress-operator default --output 'go-template={{.status.domain}}' | sed -e 's/^apps\./mk./')
+oc process -f yamls/mk-ingresscontroller.yaml DOMAIN=${domain} REPLICAS=3 | oc apply -n openshift-ingress-operator -f -
+```
+
+== Validate
+
+1. Run the following command and ensure that the ingresscontroller is reported as existing.
+
+```oc get ingresscontroller -n openshift-ingress-operator mk-ingresscontroller```
```
Comment (StevenTobin) [inline sops/sharded_ingresscontroller.asciidoc:43]: This doesn't seem to render correctly.

Comment (StevenTobin) [inline sops/sharded_ingresscontroller.asciidoc:43]: The name of the created resource is sharded so this command should be 

`oc get ingresscontroller -n openshift-ingress-operator sharded`

Comment (k-wall) [reply sops/sharded_ingresscontroller.asciidoc]: Fixed.

Comment (k-wall) [reply sops/sharded_ingresscontroller.asciidoc]: Done

**Comment by k-wall** [comment]
@StevenTobin Review comments addressed, thanks for the feedback.

**Pull request merged**